CRM/Contribute/Form/ContributionBase.php

```diff
@@ -365,7 +365,6 @@ public function preProcess() {
 
       // this avoids getting E_NOTICE errors in php
       $setNullFields = [
-        'amount_block_is_active',
```

Comment:
Since the function to interact with this handles the possibility of it not being set we don't need this enotice prevention